ivy/functional/ivy/data_type.py

```diff
@@ -231,18 +231,83 @@ def finfo(type: Union[ivy.Dtype, str, ivy.Array, ivy.NativeArray]) -> Finfo:
     -------
     ret
         an object having the followng attributes:
+    
         - **bits**: *int*
+    
           number of bits occupied by the floating-point data type.
+    
         - **eps**: *float*
+    
           difference between 1.0 and the next smallest representable floating-point
           number larger than 1.0 according to the IEEE-754 standard.
+    
         - **max**: *float*
+    
           largest representable number.
+    
         - **min**: *float*
+    
           smallest representable number.
+    
         - **smallest_normal**: *float*
+    
           smallest positive floating-point number with full precision.
 
+    This function conforms to the `Array API Standard
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.data_type_functions.can_cast.html>`_ # noqa
```

Comment:
Hi, I referred your PR for Iinfo Docstring. I believe that the api standard link for finfo is [this](https://data-apis.org/array-api/latest/API_specification/generated/signatures.data_type_functions.finfo.html)

Reply:
Hi, sorry I must have left the wrong link. Thank you for letting me notice, I will change it in my next PR

Reply:
No worries. 😊